src/docs/get-started/flutter-for/android-devs.md

```diff
@@ -53,13 +52,19 @@ For example, on iOS, you can use the [Cupertino widgets](/docs/reference/widgets
 to produce an interface that looks like
 [Apple's iOS design language](https://developer.apple.com/design/resources/).
 
+{{site.alert.secondary}}
+How is react-style, or _declarative_, programming different than imperative?
```

Comment:
I'd qualify it with `How is react-style, or _declarative_, programming different than the traditional imperative style?` since the reader might not realize 'the normal style' they do from day to day is imperative.